app/classes/query/initializers/observation_filters.rb

```diff
@@ -3,67 +3,40 @@ module Initializers
     # Handles user content-filtering parameters common to all observation
     # queries and their coerced equivalents for image, location and name.
     module ObservationFilters
-      include ::ContentFilter
-
-      # Hash of filter_parameters, each pair like:  has_images?: :string
-      # Uses filter definitions in ContentFilter to populate hash.
       def observation_filter_parameter_declarations
-        observation_filter_keys.each_with_object({}) do |f_sym, decs|
-          decs["#{f_sym.to_s}?".to_sym] = :string
-        end
-      end
-
-      # Lets application controller easily check if we need to apply user's content
-      # filter parameters to the current query.
-      def observation_filter_input
-        true
-      end
-
-      # Lets ApplicationController check whether to add default filters to
-      # current query; defaults are added unless query already has
-      # observation filter params, even if those filters are off.
-      # This allows for overriding of default filters.
-      def has_obs_filter_params?
-        observation_filter_keys.any? {|k| params[k] != nil}
+        {
+          has_specimen?: :boolean,
```

Comment:
Shouldn't this be leveraging ContentFilter.observation_filters() or a related method?